hclwrite/format.go

```diff
@@ -452,9 +457,12 @@ func tokenBracketChange(tok *Token) int {
 //
 // lead: always present, representing everything up to one of the others
 // assign: if line contains an attribute assignment, represents the tokens
-//    starting at (and including) the equals symbol
+//
+//	starting at (and including) the equals symbol
+//
 // comment: if line contains any non-comment tokens and ends with a
-//    single-line comment token, represents the comment.
+//
+//	single-line comment token, represents the comment.
```

Comment:
It seems like the modern `go fmt` misunderstood this weird extra indent as an attempt to introduce a preformatted text block.

I honestly don't recall why there were extra spaces at the start of both of these lines -- I guess maybe it was an attempt to make it clearer that this is a continuation of the definition on the previous line -- but unfortunately everything about how this is formatted disagrees with the modern godoc comment conventions, since it would all get understood as one long comment anyway.

Since this comment is attached to something that isn't exported it's kinda annoying that `go fmt` is applying the godoc formatting to it anyway, but rather than fighting that I guess we should just embrace it, and so perhaps the best answer here is to format this using the convention that godoc would expect for an unordered list, which I think would look like this:

```go
//  - lead: always present, representing everything up to one of the others
//  - assign: if line contains an attribute assignment, represents the tokens starting at (and including) the equals symbol
//  - comment: if line contains any non-comment tokens and ends with a single-line comment token, represents the comment.
```

I might not have remembered correctly how godoc prefers this list item convention to be indented, but I expect that the above would be enough to get `go fmt` to add or remove whatever whitespace it deems appropriate from before those dashes.